plugins/woocommerce/includes/admin/class-wc-admin.php

```diff
@@ -7,6 +7,8 @@
  * @version  2.6.0
  */
 
+declare(strict_types=1);
```

Comment:
Is that intentional? I have a concern that in such an old class we may run into issues because some variables in WP tend to be sometimes string sometimse an integer...

Reply:
Yes... but we don't have any type-hint here, so it should be fine.